exporter/clickhousetracesexporter/clickhouse_exporter.go

```diff
@@ -179,18 +179,15 @@ func populateOtherDimensions(attributes pcommon.Map, span *Span) {
 			if err == nil && statusString != 0 {
 				statusInt = int64(statusString)
 			}
-			span.HttpCode = strconv.FormatInt(statusInt, 10)
-			span.ResponseStatusCode = span.HttpCode
+			span.ResponseStatusCode = strconv.FormatInt(statusInt, 10)
 		} else if (k == "http.url" || k == "url.full") && span.Kind == 3 {
 			value := v.Str()
 			valueUrl, err := url.Parse(value)
 			if err == nil {
 				value = valueUrl.Hostname()
 			}
-			span.ExternalHttpUrl = value
 			span.HttpUrl = v.Str()
```

Comment:
These hold two different values. The `httpUrl` is a full URL including query params while the `ExternalHttpUrl` is just a hostname. While doing aggregation on external calls having the endpoint without query params is more important. I think `httpHost` can be a substitute, can you confirm this once.

Reply:
I couldn't find `http.host` in otel spec. I found out it is deprecated: https://github.com/open-telemetry/opentelemetry-specification/pull/2469#issue-1192460143

Reply:
Updated httpHost
https://github.com/SigNoz/signoz-otel-collector/pull/321/commits/94d43dc099924ca6ad223853f9d0a6d6bc6cebb9